docs/operations-guide/running-metabase-on-debian.md

```diff
@@ -180,14 +180,39 @@ Now, it's time to register our Metabase service with `systemd` so it will start
     # ensure unprivileged deploy user owns log (or it won't be able to write)
     $ sudo chown <your_deploy_user>:<your_deploy_user> /var/log/metabase.log
 
-    # add to default services
-    $ sudo update-rc.d metabase defaults
+#### Create a service  systemd unit file
```

Comment:
I think this sentence could be improved, e.g. _Create a systemd service_. There's also a double space now.